cmd/deploy.go

```diff
@@ -80,15 +80,17 @@ func deploy(ctx *cmdutil.Ctx) error {
 
 func generateActions(ctx *cmdutil.Ctx) (map[string]file.Op, error) {
 	assetsActions := map[string]file.Op{}
+	pathToChecksums := map[string]string{}
 
-	if len(ctx.Args) == 0 && !ctx.Flags.NoDelete {
```

Comment:
It looks like you have taken out this default case? This is the case where no filenames are provided, does theme deploy work if you just pass a single filename? or multiple?

Reply:
The behaviour is still the same. We had to change it since the `GetAllAssets` call to the server now needs to happen on the _no-args_ and _with-args_ variants of `deploy`, whereas before it only happened on the _no-args_ variant.

Reply:
haha, I was there for that change 🤦